src/test/java/me/karakelley/tictactoe/GameConfigurationTest.java

```diff
@@ -0,0 +1,38 @@
+package me.karakelley.tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameConfigurationTest {
+
+  private GameConfiguration gameConfiguration;
+
+  @Test
+  public void testGetPlayer1MarkerWithConfig() {
+    System.setProperty("configuration", "src/test/resources/config.json");
```

Comment:
I don't love this for the same reason I don't love `System.setIn`. Can you avoid mutating the global properties somehow?